docontrol/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "7622c46e-bade-44ce-99e1-a3529bf4fc04",
+  "app_id": "docontrol",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "SaaS Data Security - modernizing DLP and CASB to Secure SaaS Data",
+    "title": "DoControl",
+    "media": []
```

Comment:
We recommend adding a couple images to the tile - these can be screenshots of the DoControl UI showing the Datadog Actions in a workflow, for example.

Reply:
@anawishnoff I have added several categories aligned with our core competency.